### PR TITLE
refactor(mapping/theme): theme variables refactor

### PR DESCRIPTION
**Pull request by 32penkin**

### CODE REVIEW

File: packages/eva/themes/dark.json [hunk 1]
```diff
@@ -111,64 +111,64 @@
...
+
+  "text-warning-color": "$color-warning-default",
+  "text-warning-active-color": "$color-warning-active",
+  "text-warning-disabled-color": "$color-warning-disabled",
+
+  "text-danger-color": "$color-danger-default",
+  "text-danger-active-color": "$color-danger-active",
+  "text-danger-disabled-color": "$color-danger-disabled",
 
   "icon-color-default": "$color-basic-100",
```
Comment (nnixaa) [inline packages/eva/themes/dark.json]: same for icon [component]-[state/status]-[property]

Comment (32penkin) [reply packages/eva/themes/dark.json]: Done

File: packages/eva/themes/light.json [hunk 2]
```diff
@@ -111,64 +111,64 @@
...
+  "border-alternative-color-3": "$color-basic-1000",
+  "border-alternative-color-4": "$color-basic-1100",
+  "border-alternative-color-5": "$color-basic-1100",
+
+  "border-primary-color-1": "$color-primary-500",
+  "border-primary-color-2": "$color-primary-600",
+  "border-primary-color-3": "$color-primary-700",
+  "border-primary-color-4": "$color-primary-800",
+  "border-primary-color-5": "$color-primary-900",
+
```
Comment (artyorsh) [inline packages/eva/themes/light.json]:
```suggestion

"border-success-color-1": "$color-success-500",
"border-success-color-2": "$color-success-600",
"border-success-color-3": "$color-success-700",
"border-success-color-4": "$color-success-800",
"border-success-color-5": "$color-success-900",

"border-info-color-1": "$color-info-500",
"border-info-color-2": "$color-info-600",
"border-info-color-3": "$color-info-700",
"border-info-color-4": "$color-info-800",
"border-info-color-5": "$color-info-900",

"border-warning-color-1": "$color-warning-500",
"border-warning-color-2": "$color-warning-600",
"border-warning-color-3": "$color-warning-700",
"border-warning-color-4": "$color-warning-800",
"border-warning-color-5": "$color-warning-900",

"border-danger-color-1": "$color-danger-500",
"border-danger-color-2": "$color-danger-600",
"border-danger-color-3": "$color-danger-700",
"border-danger-color-4": "$color-danger-800",
"border-danger-color-5": "$color-danger-900",

```

File: packages/eva/themes/dark.json [hunk 3]
```diff
@@ -111,64 +111,64 @@
...
+  "border-alternative-color-3": "$color-basic-3000",
+  "border-alternative-color-4": "$color-basic-400",
+  "border-alternative-color-5": "$color-basic-500",
+
+  "border-primary-color-1": "$color-primary-500",
+  "border-primary-color-2": "$color-primary-600",
+  "border-primary-color-3": "$color-primary-700",
+  "border-primary-color-4": "$color-primary-800",
+  "border-primary-color-5": "$color-primary-900",
+
```
Comment (artyorsh) [inline packages/eva/themes/dark.json]:
```suggestion

"border-success-color-1": "$color-success-500",
"border-success-color-2": "$color-success-600",
"border-success-color-3": "$color-success-700",
"border-success-color-4": "$color-success-800",
"border-success-color-5": "$color-success-900",

"border-info-color-1": "$color-info-500",
"border-info-color-2": "$color-info-600",
"border-info-color-3": "$color-info-700",
"border-info-color-4": "$color-info-800",
"border-info-color-5": "$color-info-900",

"border-warning-color-1": "$color-warning-500",
"border-warning-color-2": "$color-warning-600",
"border-warning-color-3": "$color-warning-700",
"border-warning-color-4": "$color-warning-800",
"border-warning-color-5": "$color-warning-900",

"border-danger-color-1": "$color-danger-500",
"border-danger-color-2": "$color-danger-600",
"border-danger-color-3": "$color-danger-700",
"border-danger-color-4": "$color-danger-800",
"border-danger-color-5": "$color-danger-900",

```

File: packages/eva/mapping.json [hunk 4]
```diff
@@ -823,7 +823,7 @@
             },
             "status": {
               "primary": {
-                "dividerBackgroundColor": "border-color-primary-2"
+                "dividerBackgroundColor": "border-primary-color-2"
```
Comment (artyorsh) [inline packages/eva/mapping.json]: needs changes for other statuses

Comment (32penkin) [reply packages/eva/mapping.json]: Done

**Pull request merged**